src/Scene.ts

```diff
@@ -184,6 +192,30 @@ export class Scene {
       }
     });
 
+    /**
+     * Handle the third-person follow entity follow signal ('third_person_follow_entity').
+     * @param {string} entityName Name of the entity. Pass in null or an empty
+     * string to stop third-person following.
+     */
+    this.emitter.on(this.thirdPersonFollowEntityEvent, function(entityName) {
+
+      // Turn off following if `entity` is null.
+      if (entityName === undefined || entityName === null) {
+        that.cameraMode = '';
+        return;
+      }
+
+      var object = that.scene.getObjectByName(entityName);
+
+      if (object !== undefined && object !== null) {
```

Comment:
This one can be `!object`- `getObjectByName` returns undefined if the object is not found.

Reply:
I don't know Javascript/Typescript, so I'm honestly not sure.  But I'll point out that:

1. Even the code inside of ThreeJS uses `object !== undefined`: https://github.com/mrdoob/three.js/blob/03f35a96ae36507a2d234f6f0726dd64393ee49d/src/core/Object3D.js#L455
2. This is just a copy of the code from above, so I would suggest we do a separate pass/PR to clean this up if we think that is better.

Reply:
This can be done in a separate PR.